lib/identifiers/isbn.rb

```diff
@@ -29,7 +31,7 @@ class ISBN
         [\dX]             # Check digit
       )
       \b
-    /x
+    /x.freeze
     ISBN_A_REGEXP = %r{
       \b
       (?<=10\.) # Directory indicator (always 10)
```

Comment:
Out of curiosity, do you know why a lookbehind is used here?

Reply:
To ensure the `10\.` is there, but to exclude it from the matched result @epaune-alt. 
The same I am doing for prefixes. If you remove the lookbehind, you will get something like ["10\.", "rest_of_the_isbn"] instead of ["rest_of_the_isbn"]

Reply:
If you remove the lookbehind and its group you will get something like '10.978.8898392/315' that is what I am doing here https://github.com/altmetric/identifiers/pull/36/files#diff-9836461518e4eba5cfe982c1d12558bc16694a99c6709c2e61c2482f8019a7eeR145